academia/lectures/Lecture - You Don't Know How Wrong You Are/Instability of parameter estimates.ipynb

```diff
@@ -6,13 +6,17 @@
    "source": [
     "#Instability of Parameter Estimates\n",
     "\n",
-    "By Evgenia \"Jenny\" Nitishinskaya and Delaney Granizo-Mackenzie. Algorithms by David Edwards. Developed as part of Quantopian's Quantitative Finance Curriculum."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
+    "By Evgenia \"Jenny\" Nitishinskaya and Delaney Granizo-Mackenzie. Algorithms by David Edwards.\n",
```

Comment:
I actually kinda like having her name written like this, as she goes by Jenny. So could we have all the attributions be switched to include her nickname in the middle?